pkg/service/dependency.go

```diff
@@ -0,0 +1,103 @@
+package service
+
+import (
+	"context"
+
+	"entgo.io/ent/dialect/sql"
+	"k8s.io/apimachinery/pkg/util/sets"
+
+	"github.com/seal-io/walrus/pkg/dao/model"
+	"github.com/seal-io/walrus/pkg/dao/model/service"
+	"github.com/seal-io/walrus/pkg/dao/model/servicerelationship"
+	"github.com/seal-io/walrus/pkg/dao/model/servicerevision"
+	"github.com/seal-io/walrus/pkg/dao/types"
+	"github.com/seal-io/walrus/pkg/dao/types/object"
+	"github.com/seal-io/walrus/pkg/terraform/parser"
+)
+
+// GetServiceDependencyOutputsByID gets the dependency outputs of the service by service id.
+func GetServiceDependencyOutputsByID(
```

Comment:
Can we put it to dao/service.go

Reply:
This function depends on terraform/parser， put into dao will cause import cycle